databricks_cli/unity_catalog/delta_sharing_cli.py

```diff
@@ -547,6 +569,157 @@ def api_call(d):
         click.echo(mc_pretty_format(share_json))
 
 
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Add a shared model.')
+@click.option('--share', required=True,
+              help='Name of the share to update.')
+@create_common_shared_model_options
+@click.option('--json-file', default=None, type=click.Path(),
+              help="Adds a shared model based on shared data object represented in JSON file.")
+@click.option('--json', default=None, type=JsonClickType(),
+              help="Adds a shared model based on shared data object represented in JSON.")
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def add_share_model_cli(api_client, share, model, comment, json_file, json):
+    """
+    Adds a shared model.
+
+    The public specification for the JSON request is in development.
+    """
+    if (json_file is not None) or (json is not None):
+        def api_call(d):
+            if 'data_object_type' in d and d['data_object_type'] != "MODEL":
```

Comment:
This logic only applies to "TABLE" as when tables were the only thing that were able to be shared. So empty "data_object_type" will default to "TABLE".

See this: https://github.com/databricks/databricks-cli/blob/f01d58da604a5c7bc011c910f6df55492a5dc29a/databricks_cli/unity_catalog/delta_sharing_cli.py#L215